### ISSUE TITLE
Ask questions about the startup sequence and can't load presets, can't connect ps, can't generate, there are already solutions.

### ISSUE THREAD

**Issue by bubbl000** [NimaNzrii/comfyui-photoshop#217]
Regarding what many people say, the problem of not being able to connect to ps
At present, it is found that UXP must be installed in order for ps to connect with comfyui normally. Secondly, the ps version should be above the bate version.

About unable to generate pictures
Please check if the model has its own choice.
If it is not English, such as Chinese, you need to check the correspondence of the specified node name of the global connection (case correspondence, text correspondence). Normally, it is the number of steps, sampler, and scheduler. These need to be modified to Chinese.

Then the preset could not be loaded. I see many people say it is related to the language, but after I deleted the translation node plug-in, it still could not be loaded normally.
Then it can also be loaded into the preset in the case of Chinese.

So I want to know what is the startup order of comfyui and ps?
Do you need ps to start first, and then ps?

**Comment by bubbl000** [comment]
E:\comfyui\ComfyUI_windows_portable>.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build
## ComfyUI-Manager: installing dependencies done.
** ComfyUI startup time: 2024-08-29 00:24:04.898732
** Platform: Windows
** Python version: 3.11.8 (tags/v3.11.8:db85d51, Feb  6 2024, 22:03:32) [MSC v.1937 64 bit (AMD64)]
** Python executable: E:\comfyui\ComfyUI_windows_portable\python_embeded\python.exe
** Log path: E:\comfyui\ComfyUI_windows_portable\comfyui.log

Prestartup times for custom nodes:
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\rgthree-comfy
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use
   0.3 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager

Total VRAM 16379 MB, total RAM 65470 MB
pytorch version: 2.3.1+cu121
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4060 Ti : cudaMallocAsync
Using pytorch cross attention
[Prompt Server] web root: E:\comfyui\ComfyUI_windows_portable\ComfyUI\web
[ComfyUI-Easy-Use] server: v1.2.2 Loaded
[ComfyUI-Easy-Use] web root: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use\web_version/v2 Loaded
Total VRAM 16379 MB, total RAM 65470 MB
pytorch version: 2.3.1+cu121
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4060 Ti : cudaMallocAsync
### Loading: ComfyUI-Impact-Pack (V5.13.3)
### Loading: ComfyUI-Impact-Pack (Subpack: V0.6)
[Impact Pack] Wildcards loading done.
### Loading: ComfyUI-Inspire-Pack (V0.80.5)
### Loading: ComfyUI-Manager (V2.30)
### ComfyUI Revision: 2605 [bb4416dd] *DETACHED | Released on '2024-08-22'
#fix sys.stdout.isatty
json_repair## OK
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
--------------
 ### Mixlab Nodes: Loaded
ChatGPT.available True
edit_mask.available True
simple_lama_inpainting## OK
LaMaInpainting.available True
ClipInterrogator.available True
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json
PromptGenerate.available True
ChinesePrompt.available True
RembgNode_.available True
TripoSR.available
MiniCPMNode.available
 --------------
==============================================
[comfyui-nodes-docs]Loaded comfyui-nodes-docs plugin
==============================================
no module 'xformers'. Processing without...
no module 'xformers'. Processing without...
3D OpenPose Editor: Loaded
[comfyui_controlnet_aux] | INFO -> Using ckpts path: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts
[comfyui_controlnet_aux] | INFO -> Using symlinks: False
[comfyui_controlnet_aux] | INFO -> Using ort providers: ['CUDAExecutionProvider', 'DirectMLExecutionProvider', 'OpenVINOExecutionProvider', 'ROCMExecutionProvider', 'CPUExecutionProvider', 'CoreMLExecutionProvider']
DWPose: Onnxruntime with acceleration providers detected
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\failfast-comfyui-extensions\extensions
E:\comfyui\ComfyUI_windows_portable\ComfyUI\web\extensions\failfast-comfyui-extensions

[rgthree] Loaded 40 fantastic nodes.
[rgthree] NOTE: Will NOT use rgthree's optimized recursive execution as ComfyUI has changed.

WAS Node Suite: OpenCV Python FFMPEG support is enabled
WAS Node Suite Warning: `ffmpeg_bin_path` is not set in `E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui\was_suite_config.json` config file. Will attempt to use system ffmpeg binaries if available.
WAS Node Suite: Finished. Loaded 212 nodes successfully.

        "Art is the lie that enables us to realize the truth." - Pablo Picasso


Import times for custom nodes:
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\websocket_image_save.py
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-IC-Light-Native
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\cg-use-everywhere
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-photoshop
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_3dPoseEditor
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Noise-master
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Prompt-Expansion
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyUI_FrequencySeparation_RGB-HSV-main
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Lora-Auto-Trigger-Words
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_densediffusion
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-nodes-docs
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_IPAdapter_plus
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-inpaint-nodes
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-layerdiffuse
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Custom-Scripts
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_essentials
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\failfast-comfyui-extensions
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\Derfuu_ComfyUI_ModdedNodes
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-KJNodes
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Advanced-ControlNet
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\rgthree-comfy
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-IC-Light
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_smZNodes
   0.0 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Inspire-Pack
   0.1 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_segment_anything
   0.1 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BrushNet
   0.3 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager
   0.4 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use
   0.4 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-SUPIR
   1.2 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui
   2.2 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes
   2.4 seconds: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack





Starting server
To see the GUI go to: http://192.168.3.40:8188 or http://127.0.0.1:8188
To see the GUI go to: https://192.168.3.40:8189 or https://127.0.0.1:8189
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/photoswipe-lightbox.esm.min.js
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/photoswipe.min.css
FETCH DATA from: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/pickr.min.js
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/classic.min.css
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/juxtapose.min.js
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/juxtapose.css
data_path:  E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\data
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/photoswipe-lightbox.esm.min.js
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/photoswipe.min.css
FETCH DATA from: E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/pickr.min.js
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/classic.min.css
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/juxtapose.min.js
E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/juxtapose.css
data_path:  E:\comfyui\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes\data
# PS: client-8y88kwejo Photoshop Connected
got prompt
model weight dtype torch.float16, manual cast: None
model_type EPS
Using pytorch attention in VAE
Using pytorch attention in VAE
Requested to load SD1ClipModel
Loading 1 new model
loaded completely 0.0 235.84423828125 True
E:\comfyui\ComfyUI_windows_portable\ComfyUI\comfy\ldm\modules\attention.py:407: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)
  out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False)
Requested to load AutoencoderKL
Loading 1 new model
loaded completely 0.0 159.55708122253418 True
Requested to load BaseModel
Loading 1 new model
loaded completely 0.0 1639.406135559082 True
100%|████████████████████████████████████████████████████████████████████████████████████| 6/6 [00:01<00:00,  3.62it/s]
# PS: render done
Prompt executed in 4.96 seconds

**Comment by bubbl000** [comment]
![image](https://github.com/user-attachments/assets/eb19fa3b-9fa2-497e-b4bc-011a518d7d6e)
![image](https://github.com/user-attachments/assets/c7ce2453-b5fb-44cc-85bd-783fc3812c6f)
Normal connection on
My boot order is comfyui fully loaded and then it's boot ps hit start and the preset doesn't load out

**Comment by bubbl000** [comment]
Adobe Photoshop 版本: 25.9.0 20240516.r.573 296f739  x64
Then the antivirus software in the background is fully turned off
